app/views/staff/training_sessions/show.html.erb

```diff
@@ -140,10 +140,9 @@
     <%= submit_tag "Update", class: 'save-button' %>
   <% end %>
 
-  <% if @user.admin? %>
```

Comment:
Probably worth keeping this, or modifying it so that only admin and the current trainer can see it